marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.test.tsx

```diff
@@ -235,8 +235,16 @@ describe('Advanced Step Settings dialog', () => {
     //https://github.com/testing-library/react-testing-library/issues/375
     //Solution in github wont work because our list for additional collection is empty to start with
 
-    fireEvent.change(getByPlaceholderText('Please enter target permissions'), { target: { value: 'permissions-changed' }});
-    expect(getByPlaceholderText('Please enter target permissions')).toHaveValue('permissions-changed');
+    fireEvent.change(getByPlaceholderText('Please enter target permissions'), { target: { value: 'data-hub-operator' }});
+    expect(getByPlaceholderText('Please enter target permissions')).toHaveValue('data-hub-operator');
+    fireEvent.blur(getByPlaceholderText('Please enter target permissions'));
+    expect(getByTestId('validationError')).toHaveTextContent('The format of the string is incorrect. The required format is role,capability,role,capability,....');
```

Comment:
Let's fix the ellipsis here, it should be three dots (...) not four. (Four dots suggests the string should end in a period, which is incorrect.) And fix it in the place that is being tested: config/messages.config.ts, line 3